etc/webapi.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<routes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../app/code/Magento/Webapi/etc/webapi.xsd">
+    <route url="/V1/crud/getById" method="GET">
+        <service class="Ziffity\Task\Api\RepositoryInterface" method="getById"/>
+        <resources>
+            <resource ref="anonymous"/>
+        </resources>
+    </route>
+    <route url="/V1/crud/search" method="GET">
+        <service class="Ziffity\Task\Api\RepositoryInterface" method="getList"/>
+        <resources>
+            <resource ref="anonymous"/>
+        </resources>
+    </route>
+    <route url="/V1/crud" method="POST">
+        <service class="Ziffity\Task\Api\RepositoryInterface" method="save"/>
+        <resources>
+            <resource ref="anonymous"/>
```

Comment:
@NandhiniKarthi Only people who have access should allowed to write. not everyone.